dmq-node/src/DMQ/Protocol/SigSubmission/Type.hs

```diff
@@ -181,11 +185,14 @@ data Sig crypto = SigWithBytes {
     -- ^ the `SigRaw` data type along with signed bytes
   }
 
-deriving instance ( DSIGNAlgorithm (KES.DSIGN crypto)
-                  , Show (VerKeyKES (KES crypto))
-                  , Show (SigKES (KES crypto))
-                  )
-               => Show (Sig crypto)
+instance Show (Sig crypto) where
+  show Sig { sigId } = "Sig crypto ... " <> show sigId
```

Comment:
We shouldn't use `Show` for, could you just leave a TODO for adding a `Display` type class.